File: Array/Easy/second largest/secLargest.js
const secLargest = arr => {
  let max = arr[0]
  let secMax = arr[0]
  const len = arr.length
  for (let i = 1; i < len; i++) {
    if (arr[i] > max) {
      secMax = max
      max = arr[i]
    } else if (arr[i] > secMax && arr[i] != max) {
      secMax = arr[i]
    }
  }
  console.log(secMax, max)
  return secMax
}

secLargest([2, -1, 5, 7, 3, 12, 35, -34, 18, -55])

File: Array/Easy/second largest/secLargest.py
def secLargest(list):
    max = list[0]
    secMax = list[0]

    for i in range(1,len(list)):
        if list[i] > max:
            secMax = max
            max = list[i]
        elif list[i] > secMax and list[i] != max:
            secMax = list[i]
    print(secMax)
    return secMax

secLargest([2, -1, 5, 7, 3, 12, 35, -34, 18, -55])